### PR TITLE
Switch Accumulo into multiple modules, one per minor release

### PR DESCRIPTION
**Pull request by busbey**

### CODE REVIEW

**Comment by busbey** [comment]
@madrob, @joshelser  can I get a review?

File: accumulo1.7/src/main/conf/accumulo.properties [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Copyright 2014 Cloudera, Inc. or its affiliates. All Rights Reserved.
```
Comment (joshelser) [inline accumulo1.7/src/main/conf/accumulo.properties]: Intentional?

File: accumulo1.8/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+LICENSE file.
+-->
+
+## Quick Start
+
+This section describes how to run YCSB on [Accumulo](https://accumulo.apache.org/). 
+
+### 1. Start Accumulo
+
+See the [Accumulo Documentation](https://accumulo.apache.org/1.7/accumulo_user_manual.html#_installation)
```
Comment (joshelser) [inline accumulo1.8/README.md]: Update for the 1.8 user manual?

File: accumulo1.7/src/main/conf/accumulo.properties (hunk 1 above)
Comment (busbey) [reply accumulo1.7/src/main/conf/accumulo.properties]: must be from before we switched to the "YCSB contributors" language.  not sure if copying it into a new directory is a creative act worthy of updating though.

File: accumulo1.8/README.md (hunk 2 above)
Comment (busbey) [reply accumulo1.8/README.md]: ✅

**Comment by busbey** [comment]
yeah, future proofing. I figure this will make it easier to compare perf using the 1.6 - 1.8 clients, but I have no specific reason to think they'll be different.

**Comment by busbey** [comment]
I pushed an update that switched the 1.8 and 1.6 specific bindings to both refer to their respective version's user manual.

**Comment by busbey** [comment]
planning to push this based on @joshelser's previous +1 later today. speak up if anyone would like more time to review further.

**Comment by joshelser** [comment]
+1 from me. Thanks Sean. Nice work here.

**Pull request merged**